en/theme/material/assets/css/redoc.css

```diff
@@ -137,6 +137,11 @@
     color: #fff !important;
 }
 
+.api-content td div[class="sc-ikkxIA daqcVd"] {
```

Comment:
Are we sure that these class names wouldn't change between builds?

Reply:
I tried multiple times building the docs but class name wasn't changed.

Reply:
Additionaly I have verified building in different computer and it worked as expected.